src/Auth0.OidcClient.Core/Auth0ClientOptions.cs

```diff
@@ -19,11 +19,6 @@ public class Auth0ClientOptions
         /// </summary>
         public string ClientId { get; set; }
 
-        /// <summary>
-        /// Your Auth0 Client Secret.
-        /// </summary>
-        public string ClientSecret { get; set; }
```

Comment:
I'm not seeing any milestone set on this PR, and given that is being submitted against `master` I assume is for the "current version". Isn't removing this getter/setter a breaking change?

Reply:
This is for v3, the rest of which including many other breaking changes is in master waiting for this and #110 to release.

Haven't been using milestones on this repo.